test/README.md

```diff
@@ -0,0 +1,82 @@
+# Cluster Version Operator Tests
+
+We hold all e2e tests and related common methods in this folder.
+Test script are using client-go, here is the document: https://pkg.go.dev/k8s.io/client-go
```

Comment:
I do not think we need to mention the dependency of the test, unless there are something special for this particular one: `client-go`.

Reply:
This description is helpful for new contributors who are unfamiliar with client-go.